bin/provision.sh

```diff
@@ -221,8 +221,9 @@ else
 	exit 1;
 fi
 
-echo "export PS1='\[\033[1;33m\]OCdev:\[\033[m\w\$ '" >> /home/vagrant/.bashrc
-echo "cd /var/www/html/htdocs/" >> /home/vagrant/.bashrc
+echo "export PS1='\[\033[38;5;11m\]OCdev:\[$(tput sgr0)\]\[\033[38;5;15m\] \[$(tput sgr0)\]\[\033[38;5;14m\]\w\[$(tput sgr0)\]\[\033[38;5;15m\]\\$ \[$(tput sgr0)\]'" >> /home/vagrant/.bashrc
```

Comment:
this is not in the scope of this PR, but the old one scrambles the input some times if you use the "up" key